### PR TITLE
AMQ-5503: Improve initscript, reintegrate changes

### PR DESCRIPTION
**Pull request by scoopex**
https://issues.apache.org/jira/browse/AMQ-5503

### CODE REVIEW

File: assembly/src/release/bin/activemq [hunk 1]
```diff
@@ -109,14 +96,7 @@ if [ -z "$ACTIVEMQ_BASE" ] ; then
...
 
-# Configure user specified classpath here or externally using this variable
-if [ -z "$ACTIVEMQ_USER_CLASSPATH" ] ; then
-    ACTIVEMQ_USER_CLASSPATH=""
-fi
-
-# ActiveMQ Classpath configuration
-ACTIVEMQ_CLASSPATH="$ACTIVEMQ_BASE/../lib/:$ACTIVEMQ_USER_CLASSPATH"
-
+ACTIVEMQ_CLASSPATH="$ACTIVEMQ_BASE/../lib/"
```
Comment (jbonofre) [inline assembly/src/release/bin/activemq]: The user should be able to define its classpath extend, no ? What's the rationale here ?

File: assembly/src/release/bin/activemq [hunk 2]
```diff
@@ -325,20 +305,16 @@ invokeJar(){
...
-      fi
       $EXEC_OPTION $DOIT_PREFIX "\"$JAVACMD\" $ACTIVEMQ_OPTS $ACTIVEMQ_DEBUG_OPTS \
               -Dactivemq.classpath=\"${ACTIVEMQ_CLASSPATH}\" \
               -Dactivemq.home=\"${ACTIVEMQ_HOME}\" \
               -Dactivemq.base=\"${ACTIVEMQ_BASE}\" \
               -Dactivemq.conf=\"${ACTIVEMQ_CONF}\" \
               -Dactivemq.data=\"${ACTIVEMQ_DATA}\" \
               $ACTIVEMQ_CYGWIN \
-              -jar \"${ACTIVEMQ_HOME}/bin/activemq.jar\" $COMMANDLINE_ARGS >> $ACTIVEMQ_OUT 2>&1 &
+              -jar \"${ACTIVEMQ_HOME}/bin/activemq.jar\" $COMMANDLINE_ARGS >/dev/null 2>&1 &
```
Comment (jbonofre) [inline assembly/src/release/bin/activemq]: User can still be able to redirect output to a file as he wants.

File: assembly/src/release/bin/activemq [hunk 3]
```diff
@@ -386,8 +362,8 @@ checkRunning(){
         echo "ERROR: Pidfile '$ACTIVEMQ_PIDFILE' exists but contains no pid"
         return 2
        fi
-       ACTIVEMQ_PID="`cat ${ACTIVEMQ_PIDFILE}`"
-       RET="`ps -p "${ACTIVEMQ_PID}"|grep java`"
+       ACTIVEMQ_PID="`cat $ACTIVEMQ_PIDFILE`"
+       RET="`ps -p "$ACTIVEMQ_PID"|grep java`"
```
Comment (jbonofre) [inline assembly/src/release/bin/activemq]: It's more portable to keep the genere ${xx} format IMHO.

File: assembly/src/release/bin/activemq (hunk 1 above)
Comment (scoopex) [reply assembly/src/release/bin/activemq]: No, that doesn't seem to be necessary from my point of view.

File: assembly/src/release/bin/activemq (hunk 3 above)
Comment (scoopex) [reply assembly/src/release/bin/activemq]: from my point of view this doesn't make a huge difference.

**Comment by lucastetreault** [comment]
Can one of the admins verify this patch?

**Comment by jbonofre** [comment]
No update from contributor.

**Pull request closed without merging**